testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -81,9 +68,28 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I see "SUSE Linux Enterprise Server 15 SP2 x86_64" product has been added
     Then the SLE15 SP2 product should be added
 
+@scc_credentials
+  Scenario: Add a product with recommended enabled
```

Comment:
I think this will break the autoinstallation scenario at `minkvm_guests.feature`: we are installing there a SLE 15 SP2.
Please, ask to @cbosdo if we can update the KVM test to use 15 SP3 or if we should stick to 15 SP2 instead.

Reply:
Why that could be a problem? We are syncing SP2 in the previous scenario.